source/partials/next_release/_release.md.erb

```diff
@@ -1,87 +1,88 @@
-#### API Enhancements
+<h4>Authorization Plugin</h4>
 
-* <%= link_to_commit 'f47d054171b07e2ce9c2efa9e02dde6979fc9c8f', "#{link_to_api 'encryption'} to encrypt secrets.", 1829 %>
+There has been more work done around fine tuning the Authorization Plugin endpoints, as part of this some of the plugin API calls have changed.
```

Comment:
Needs an "and" -> There has been more work done around fine tuning the Authorization Plugin endpoints and as part of this some of the plugin API calls have changed.